diff --git a/math-libs/BLAS/CMakeLists.txt b/math-libs/BLAS/CMakeLists.txt
--- a/math-libs/BLAS/CMakeLists.txt
+++ b/math-libs/BLAS/CMakeLists.txt
@@ -60,5 +60,6 @@ if(_enable_rocRoller)
       -DHIP_PLATFORM=amd
       -DROCROLLER_ENABLE_YAML_CPP=ON
      -DROCROLLER_ENABLE_CLIENT=OFF
      -DROCROLLER_BUILD_TESTING=${THEROCK_BUILD_TESTING}
+      -DROCROLLER_ENABLE_GEMM_CLIENT_TESTS=OFF
      -DROCROLLER_BUILD_SHARED_LIBS=ON
